actions/lib/ansible_base.py

```diff
@@ -21,6 +22,22 @@ def __init__(self, args):
         :type args: ``list``
         """
         self.args = args[1:]
+
+        # handle multiple variables in extra_vars list
+        for i, arg in enumerate(self.args):
```

Comment:
You're right- I'm going to move this out into a method with a better name and just call it from the constructor.